### ISSUE TITLE
missing .Rdata object after run

### ISSUE THREAD

**Issue by traumenCapote** [JinmiaoChenLab/cytofkit2#17]
Hi, I have run cytofKit2 for the first time and the Rdata file is nowhere to be seen. Aside from some errors at the end, it does state that the file is saved but I cannot find it. This is the output at the end of the run:

Analysis DONE, saving the results...
R object is saved in  XXX.RData 
  **THIS R OBJECT IS THE INPUT OF SHINY APP!**  
Warning: Removed 565917 rows containing missing values (geom_point).
Warning: Error in if: missing value where TRUE/FALSE needed
  98: translate_shape_string
  97: f
  96: g$draw_key
  95: FUN
  94: lapply
  93: FUN
  92: lapply
  90: guide_gengrob.legend
  88: lapply
  87: guides_gengrob
  86: build_guides
  85: ggplot_gtable.ggplot_built
  83: print.ggplot
  81: grid.draw.ggplot
  79: ggsave
  78: cytof_writeResults
  77: cytofkit
  73: observeEventHandler [/Library/Frameworks/R.framework/Versions/3.6/Resources/library/cytofkit2/shiny/server.R#71]
   2: shiny::runApp
   1: cytofkit_shiny_dashboard

**Comment by grhogg** [comment]
Yes, I have this issue as well when running large numbers of FCS files, but it doesn't occur when running smaller batches of FCS files.

**Comment by traumenCapote** [comment]
> Yes, I have this issue as well when running large numbers of FCS files, but it doesn't occur when running smaller batches of FCS files.

So just to be clear, can you tell for sure that this is caused by the number of fcs files (which I realize is what you said) and not the number of cells (which would correlate with file number)? If the former, then I can just concatenate all files into one large one. FYI, I was running 90-ish fcs files.

**Comment by grhogg** [comment]
Sorry, I haven't looked into this yet, but you are right, there could be a limit to either fcs files or cell count.

**Comment by grhogg** [comment]
I just tried with all the FCS files together but a very small cell number (50/file) and it still fails to return any data, so I believe it is indeed the number of fcs files

**Comment by roelar** [comment]
Is there any solution for this issue? I am experiencing the same problem with large number of fcs files (about 35). Thank you!

**Comment by HoxB8** [comment]
> > Yes, I have this issue as well when running large numbers of FCS files, but it doesn't occur when running smaller batches of FCS files.
> 
> So just to be clear, can you tell for sure that this is caused by the number of fcs files (which I realize is what you said) and not the number of cells (which would correlate with file number)? If the former, then I can just concatenate all files into one large one. FYI, I was running 90-ish fcs files.

Hello, I'm having this same issue.  Have you tried concatenation to see if it worked for you?
